odxtools/parameters/tablekeyparameter.py

```diff
@@ -91,6 +91,7 @@ def _resolve_odxlinks(self, odxlinks: OdxLinkDatabase) -> None:
                 self._table_row = odxlinks.resolve(self.table_row_ref, TableRow)
             else:
                 self._table_row = odxlinks.resolve(self.table_row_ref)
+            self._table_row._resolve_odxlinks(odxlinks)
```

Comment:
Without this line, the parser will raise exception on below line, said that the "table" of tablerow does not exist.

Reply:
I see. That said, `_resolve_odxlinks()` must not be called for objects that are the result of a reference resolution lest there be cyclic reference resolution hell. as an alternative to this patch line 94 should better be replaced by
```suggestion(
            self._table = odxlinks.resolve(self._table_row.table_ref, Table)
```

Reply:
do you want you change the PR to only fix this issue (and remove the SNREF changes) or do you prefer if I do it?